PencilCase/PencilCase/scripts/application.ts

```diff
@@ -86,6 +87,86 @@ export class Application {
         this.initializeDatabase();
     }
 
+
+    private enableFilterableSelect = () => {
```

Comment:
For the implementation here, refer to the JQM demo here: http://demos.jquerymobile.com/1.4.5/selectmenu-custom-filter/